<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormJockeys</class>
 <widget class="QWidget" name="FormJockeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="mainName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MathJax_Fraktur</family>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SKachKi</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="name">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="surname">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="age">
       <property name="text">
        <string>Возраст</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p1">
       <property name="text">
        <string>1 место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p2">
       <property name="text">
        <string>2 место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p3">
       <property name="text">
        <string>3 место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="moreInfoCol">
       <property name="text">
        <string>Подробности</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>928</width>
        <height>435</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
